runtime/src/bank/tests.rs

```diff
@@ -2297,32 +2297,40 @@ fn test_verify_snapshot_bank() {
 
 // Test that two bank forks with the same accounts should not hash to the same value.
 #[test]
-fn test_bank_hash_internal_state_same_account_different_fork() {
+fn test_bank_hash_same_account_different_fork() {
```

Comment:
```suggestion
// Test that two bank forks with the same transactions should not hash to the same value.
#[test]
fn test_bank_hash_same_transactions_different_fork() {
```
I think this is more accurate considering that the accounts are actually different (because of the clock sysvar)